### PR TITLE
Public Release for Fall24 meta-release

### LINKED ISSUE #69: Convention for file names of API definition
**Problem description**
There is currently no guidelines for file names of API definition files in CAMARA.
The initially agreed proposal is to use the convention `api-name.yaml` as a guideline for the API definition file names.

**Expected behavior**
Change the file name to `home-devices-qod.yaml` before or as a part of release PR for M4.

**Alternative solution**
Wait for the final guidelines.

**Additional context**
See the discussion in [Commonalities](https://github.com/camaraproject/Commonalities/issues/284)

### PR DESCRIPTION
**Pull request by jpengar**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

Public release (r1.2) for 0.4.0 version of the HomeDevicesQoD API.

#### Which issue(s) this PR fixes:

Fixes #69 
Related to https://github.com/camaraproject/HomeDevicesQoD/issues/63

#### Special notes for reviewers:

N/A

#### Changelog input

```
 Public release of home-devices-qod v0.4.0
```

#### Additional documentation 

https://wiki.camaraproject.org/display/CAM/Meta-release+Fall24
https://wiki.camaraproject.org/display/CAM/home-devices-qod+v0.4.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +16,56 @@ The below sections record the changes for each API version in each release as fo
...
+* N/A
+
+### Changed
+
+* Update and review codeowners and maintainers md file by @hdamker in https://github.com/camaraproject/HomeDevicesQoD/pull/62
+* Update README.md according to camaraproject/Template_Lead_Repository by @jpengar in https://github.com/camaraproject/HomeDevicesQoD/pull/67
+* API spec update for Fall24 meta-release by @jpengar in https://github.com/camaraproject/HomeDevicesQoD/pull/64
+* Update testing definitions in .feature file for Fall24 meta-release by @jpengar in https://github.com/camaraproject/HomeDevicesQoD/pull/65
+* Update with the new API-Readiness-Checklist.md for Fall24 meta-release by @jpengar in https://github.com/camaraproject/HomeDevicesQoD/pull/66
+* Update filenames to kebad-case format by @jpengar in https://github.com/camaraproject/HomeDevicesQoD/pull/70
```
[the remark this answers is not in the document]
Comment (hdamker) [reply CHANGELOG.md:54]: And then you could also omit me as a contributor :-)

File: documentation/API_documentation/home-devices-qod-API-Readiness-Checklist.md
[the remark this answers is not in the document]
Comment (jpengar) [reply documentation/API_documentation/home-devices-qod-API-Readiness-Checklist.md:1]: Fixed in https://github.com/camaraproject/HomeDevicesQoD/pull/70/commits/dc2a578bfb263a2588e0ed2a9ecb154d75ba9ffc

**Comment by hdamker** [comment]
Thanks a lot @jpengar. Now we have a poster child for a great release PR :-) 
cc: @bigludo7

**Review by fernandopradocabrillo — APPROVED** [review]
LVGTM

**Comment by hdamker** [comment]
Just to avoid misunderstanding: please wait for with the merge of this PR the official approval from Release Management .

**Comment by jpengar** [comment]
> Just to avoid misunderstanding: please wait for with the merge of this PR the official approval from Release Management .

@hdamker Of course, I didn't plan to merge the PR before release management approval.

**Review by bigludo7 — APPROVED** [review]
Approved on behalf of the release-management_maintainers

**Pull request merged**